src/snovault/elasticsearch/indexer_state.py

```diff
@@ -38,6 +46,100 @@ def includeme(config):
     config.add_route('_indexer_state', '/_indexer_state')
     config.scan(__name__)
 
+
+def _get_this_instance_name(instance_name=None):
```

Comment:
I'm wondering if this is really the best place for this function, it seems pretty generic. Just a thought, I'm not opposed to leaving it here since it's only used for indexing.